internal/website/themes/hyde/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
It's annoying to have all this theme stuff in here. Isn't there something we can do to only have the real content?

Reply:
I'm not sure what. We could extract only the essential parts of the theme - that would reduce the line count but would require significant time investment. The theme was modified for our needs so it's not something we can just link to outside, and it's essential for Hugo to generate the HTML.